_shared/project/Makefile

```diff
@@ -159,7 +135,7 @@ sure:
 # requirements/%.txt filename, for example requirements/foo.txt -> foo.
 requirements/%.txt: requirements/%.in
 	@touch -a $(subst prod.txt,dev.txt,$@)
-	@tox -qe $(subst prod,dev,$(basename $(notdir $@))) --run-command 'pip --quiet --disable-pip-version-check install pip-tools'
+	@tox -qe $(subst prod,dev,$(basename $(notdir $@))) --run-command 'pip --quiet --disable-pip-version-check install pip-tools pip-sync-faster'
```

Comment:
I ran into an issue with `make requirements` that I somehow hadn't come across before:

1. Create a new project using the cookiecutter. At first the new project will get starter `requirements/*.in` files from the cookiecutter but there won't be any `requirements/*.txt`'s yet at first
2. Run `make requirements` (the cookiecutter does this automatically for you when you create a new project). `make requirements` first creates empty `requirements/*.txt` files (using `touch`) which is necessary because [tox tries to install the `requirements/*.txt` file](https://github.com/hypothesis/cookiecutters/blob/fb5b255cbc4179a49fd4c6ee373d4883e7d060ef/_shared/project/tox.ini#L59) and will crash if it doesn't exist. `make requirements` then runs `pip install pip-tools` which is necessary because `pip-compile` needs to be installed in the fresh, empty venv that tox has just created. Then finally `make requirements` runs `pip-compile` and replaces the empty `requirements/*.txt` file with the compiled one.
3. Now try to run `make sure` (or `make test` or anything). The first thing that will happen is that [tox will try to run `pip-sync-faster`](https://github.com/hypothesis/cookiecutters/blob/fb5b255cbc4179a49fd4c6ee373d4883e7d060ef/_shared/project/tox.ini#L95) which will fail because `pip-sync-faster` isn't installed in the virtualenv. `pip-sync-faster` **is** listed in all the `requirements/*.in` files (the cookiecutter makes sure of that) and it **is** also listed in all the `requirements/*.txt` files now that they've been compiled. But `pip-sync-faster` is not installed in the virtualenv yet because the `requirements/*.txt` file hasn't been installed in the virtualenv yet because that's `pip-sync-faster`'s job! So we have a circular problem.

Anyway, the solution was just to have `make requirements` install `pip-sync-faster` if it isn't already installed.